code/game/objects/machinery/doors/airlock.dm

```diff
@@ -84,13 +86,36 @@
 		if(24 to 30)
 			machine_stat ^= PANEL_OPEN
 
+/obj/machinery/door/airlock/proc/cyclelinkairlock()
```

Comment:
autodoc

Reply:
This one is done, it's just not showing here for some reason